### PR TITLE
Adds a damage multiplier var for akimbo, nerfs revolver akimbo firing delay

### PR DESCRIPTION
**Pull request by lKiseki**
## About The Pull Request
Adds a damage multiplier for akimbo. This is a var in every gun, and is set to 0.75 by default (meaning all akimbo damage gets a -25% damage modifier).
Additionally, nerfs the akimbo firing delay for revolvers.

## Why It's Good For The Game
Akimbo has a lot of damage in general, and a lot of firing speed on revolvers. This PR aims to amend that by tackling the damage output akimbo can generate, as well as introducing a firing speed nerf for dual wielding revolvers.

## Changelog
:cl: Lewdcifer
code: All guns now have a damage multiplier var for akimbo.
balance: All guns will now receive a -25% damage modifier when dual wielded.
balance: Revolvers now have an additional delay of 80 milliseconds.
/:cl:

### CODE REVIEW

**Comment by Pariah919** [comment]
Haven't we honestly nerfed this playstyle enough? I don't think we need a flat nerf.

**Comment by lKiseki** [comment]
> Haven't we honestly nerfed this playstyle enough? I don't think we more flat nerfs.

Not enough if it's still deleting xenos in nearly a blink of an eye.

**Comment by Strayed-Raven** [comment]
Who would win. Maints or a playstyle? 😂

**Comment by Strayed-Raven** [comment]
> Haven't we honestly nerfed this playstyle enough? I don't think we more flat nerfs.
Easy when a playstyle isnt 'intended'
Who would win. Maints or a playstyle? 😂

**Comment by Xsdew** [comment]
Is it the revolver being akimbo itself or the revolver spam being an issue? 

Because to kill the spam you need to kill Quick Equip working on bags.

**Comment by lKiseki** [comment]
> Is it the revolver being akimbo itself or the revolver spam being an issue?
> 
> Because to kill the spam you need to kill Quick Equip working on bags.

Eh... I don't really mind the spam too much compared to the sheer damage output for akimbo and the lack of an akimbo firing delay on revolvers. I commend peeps for what they do with several bags and whatnot of guns, but I don't think they should be rewarded as much as they currently are.

**Comment by Pariah919** [comment]
> > Is it the revolver being akimbo itself or the revolver spam being an issue?
> > Because to kill the spam you need to kill Quick Equip working on bags.
> 
> Eh... I don't really mind the spam too much compared to the sheer damage output for akimbo and the lack of an akimbo firing delay on revolvers. I commend peeps for what they do with several bags and whatnot of guns, and I don't mind that, but I don't think they should be rewarded as much as they currently are.

Most people who I see do well with this playstyle don't even akimbo it. You're applying a flat nerf on every single gun when akimbo'ed which will have very far reaching effects, nerfing meme tier stuff like double rifles akimbo'ed and other chaff loadouts seems like a bad decision to make when revolvers are the main issue to you.

**Comment by lKiseki** [comment]
> > > Is it the revolver being akimbo itself or the revolver spam being an issue?
> > > Because to kill the spam you need to kill Quick Equip working on bags.
> > 
> > 
> > Eh... I don't really mind the spam too much compared to the sheer damage output for akimbo and the lack of an akimbo firing delay on revolvers. I commend peeps for what they do with several bags and whatnot of guns, and I don't mind that, but I don't think they should be rewarded as much as they currently are.
> 
> Most people who I see do well with this playstyle don't even akimbo it. You're applying a flat nerf on every single gun when akimbo'ed which will have very far reaching effects, nerfing meme tier stuff like double rifles akimbo'ed and other chaff loadouts seems like a bad decision to make when revolvers are the main issue to you.

I never said revolvers were the main issue. That is precisely why this is being applied to every gun in existence. Revolvers just have another issue with their firing speed on akimbo, which I decided to tackle in the same PR.

File: code/modules/projectiles/guns/revolvers.dm [hunk 1]
```diff
@@ -16,6 +16,7 @@
 	actions_types = list(/datum/action/item_action/aim_mode)
 	aim_speed_modifier = 0.75
 	aim_fire_delay = 0.25 SECONDS
+	akimbo_additional_delay = 0.08 SECONDS
```
[the remark this answers is not in the document]
Comment (lKiseki) [reply code/modules/projectiles/guns/revolvers.dm]: I'm used to seconds, the value is still what's intended.

**Comment by GreatKingWombat** [comment]
Isn't 25% a little much? Will this end up nerfing akimbo so much that you deal more damage with just a single weapon?

**Comment by lKiseki** [comment]
> Isn't 25% a little much? Will this end up nerfing akimbo so much that you deal more damage with just a single weapon?

No, that'd happen if the modifier were -50%. A -25% modifier ensures that there's still a very tangible gain from dual wielding two guns, it just won't be as crazy as it is currently.

**Comment by Al-1ce** [comment]
Akimbo remains stupid dangerous and has already been nerfed a dozen times.

-1 from me.

**Comment by SilencedMP5A5** [comment]
In the arms of the angel
Fly away from here
From this dark cold hotel room
And the endlessness that you fear
You are pulled from the wreckage
Of your silent reverie
You're in the arms of the angel

**Comment by Naaanii** [comment]
Guns needing to be akimbod are flawed in themselves so this kind of works. If quick draw is used, it could be addressed later but this is probably the best stepping stone to make such weapons actually have a single wield niche instead of having to just dual wield.

Akimbo dps is higher but also has shit scatter and accuracy, the main issue is that this DPS is dished out at melee range. With scatter and other mechanics, I don't think it's too much of an issue if Xenos play around it but I don't think this is a bad PR either.

**Pull request closed without merging**